src/mocks/handlers/labwareHandlers.ts

```diff
@@ -93,6 +96,24 @@ const labwareHandlers = [
         })
       );
     }
+  ),
+
+  graphql.query<GetSuggestedWorkForLabwareQuery, GetSuggestedWorkForLabwareQueryVariables>(
+    'GetSuggestedWorkForLabware',
+    (req, res, ctx) => {
+      const work = workFactory.build({ workNumber: 'SGP1008' });
+      return res(
+        ctx.data({
+          suggestedWorkForLabware: {
+            suggestedWorks: [...req.variables.barcodes].map((barcode) => ({
+              workNumber: work.workNumber,
+              barcode: barcode
+            })),
+            works: [work]
```

Comment:
I think `works:` isn't needed, in line with the last change

Reply:
Yes, updated